test/regressions/.mocharc.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   recursive: true,
+  slow: 500,
   timeout: (process.env.CIRCLECI === 'true' ? 4 : 2) * 1000, // Circle CI has low-performance CPUs.
-  reporter: 'dot',
```

Comment:
We don't use the spec reporter because [you didn't want it](https://github.com/mui-org/material-ui/pull/22924#issuecomment-707366728). Let's do this in a different PR and restore the spec reporter for the other tests as well.

We should not use different reporters for tests.

Reply:
Thanks for linking the initial review. The motivation here to make it easier to spot slow tests:

<img width="738" alt="Screenshot 2021-02-21 at 15 00 25" src="https://user-images.githubusercontent.com/3165635/108627392-a4271900-7455-11eb-842f-aa8da2b24a99.png">

We still get the colored output. I didn't test if we get the context when an error occurs. 

Is the karma mocha spec reporter behaving like the naked mocha spec reporter? If we get the color **and** the context of the parent describes when it fails, I'm all in for changing it to spec.

Reply:
reverted